androbot/templates.py

```diff
@@ -0,0 +1,8 @@
+from typing import Union
+
+import aiogram.types as aiotypes
+
+
+def render_message(template_text: str, message: Union[aiotypes.Message, aiotypes.CallbackQuery]):
+    parsed_text = template_text.replace("{{username}}", message.from_user.username)
+    return parsed_text
```

Comment:
мб в шаблоне хранить `"Hello {username}"`, а потом тут можно:

```suggestion
    return template_text.format(username=message.from_user.username)
```

Reply:
А вообще, можно jinja2 использовать для шаблонов, но не уверен, надо ли..

Reply:
да тоже вот подумал, может jina2
но вроде пока все простые шалоны будут
не подумал про format почему-то, да с ним удобнее